app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/models/VendorFilter.scala

```diff
@@ -462,6 +462,30 @@ object VendorFilterGroups {
     UkInterest, ConstructionIndustryScheme, Employment, UkDividends, StatePensionIncome, PrivatePensionIncome, ForeignDividends, ForeignInterest, PaymentsIntoAPrivatePension, CharitableGiving, CapitalGainsTax, StudentLoans, MarriageAllowance, VoluntaryClass2NationalInsurance, HighIncomeChildBenefitCharge
   )
 
+  def nonMandatedIncomeGroupFiltered(partnerIncomeAllowed: Boolean, fosterCarerAllowed: Boolean, trustAllowed: Boolean): List[VendorFilter] = {
```

Comment:
I think it should be ok to change the nonMandatedIncomeGroup list as ultimately when the feature switch is on we want to be able to filter by the new filter too